docs/README.md

```diff
@@ -2,95 +2,103 @@
 
 ## Overview
 
-Crossplane Provider for IONOS Cloud gives the ability to manage IONOS Cloud infrastructure directly from Kubernetes.
-Crossplane extends a Kubernetes cluster to support orchestrating any infrastructure or managed service. Providers extend
-Crossplane to enable infrastructure resource provisioning of specific API.
+Crossplane Provider for IONOS Cloud gives the ability to manage IONOS Cloud infrastructure directly from Kubernetes. Crossplane extends a Kubernetes cluster to support orchestrating any infrastructure or managed service. Providers can use Crossplane to enable infrastructure resource provisioning of a specific API.
 
-The [crossplane-provider-ionoscloud](https://github.com/ionos-cloud/crossplane-provider-ionoscloud) repository is the
-Crossplane infrastructure provider for IONOS Cloud. The provider that is built from the source code from the repository
-can be installed into a Crossplane control plane and adds the following new functionality:
+{% hint style="info" %}
```

Comment:
This style of hints doesn't  seem to render very well in Github.

Reply:
Hi @cristiGuranIonos thanks a lot for your concern. As discussed with you and my team, we cannot change the format, because it is the standard which we follow for the consistency reasons.

For users using GitHub, they still can see the relevant tags if it is a Note or a Warning etc. So, I think it serves the purpose well.